app/build.gradle

```diff
@@ -57,7 +57,7 @@ dependencies {
         exclude module: 'support-annotations'
     })
 
-    implementation 'com.github.TeamNewPipe:NewPipeExtractor:5f65788a2f89e'
+    implementation 'com.github.TeamNewPipe:NewPipeExtractor:4488c217659d2e92'
```

Comment:
@theScrabi What about https://github.com/TeamNewPipe/NewPipeExtractor/pull/177? I've already updated to https://github.com/TeamNewPipe/NewPipeExtractor/commit/5f65788a2f89eedbf384aa5c5f3f98dcfd73400e but you downgrade to https://github.com/TeamNewPipe/NewPipeExtractor/commit/4488c217659d2e923f125fd923e1b28812cf217f.

Reply:
ups ... forgot to pull the extractor sorry